src/commands.rs

```diff
@@ -461,6 +461,41 @@ pub fn list(ctx: &mut Context<'_>, no_connect: bool) -> anyhow::Result<()> {
   Ok(())
 }
 
+/// Fill the SD card with random data
+pub fn fill(ctx: &mut Context<'_>) -> anyhow::Result<()> {
+  with_storage_device(ctx, |ctx, mut device| {
+    let pin_entry = pinentry::PinEntry::from(args::PinType::Admin, &device)?;
+
+    // Similar to reset, we want the user to re-enter the admin PIN even if is cached to avoid
+    // accidental data loss.
+    pinentry::clear(&pin_entry).context("Failed to clear cached secret")?;
+
+    try_with_pin(ctx, &pin_entry, |pin| {
+      device.fill_sd_card(&pin).context("Failed to fill SD card")
+    })?;
+
+    let mut last_progress = 0;
+    loop {
```

Comment:
Should we add some form of delay in here? Seems not wise to effectively busy loop for an hour. Granted, most of the time we will probably be waiting for the card but still...given how long the operation takes it seems a second here or there doesn't matter :D

Reply:
The final version has a one second delay between calls.  I think that’s okay.

Reply:
Maybe it could be increased to 10 seconds if we assume that the operation takes 60 minutes and that the progress increases linearly with time.

Reply:
I was just worried about a busy loop, but a 1s delay is quite a long time for a computer.